setup.py

```diff
@@ -77,6 +78,7 @@
     extras_require={
         "tests": tests_require + server_require,
         "server": server_require,
+        "locales": server_locales_require,
```

Comment:
This is meant to be used with `[server]` (i.e. `pipx install buku[server,locales]`).

I haven't found any good way to define a "combo" (so that using `[locales]` without `[server]` doesn't pull Flask-Admin)… should still be fine, I guess :thinking: